content/en/acknowledgements.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Acknowledgements
```

Comment:
Interesting idea!

Reply:
I would put this page under `/docs`, with a heavy `weight` so that it appears at the bottom of the left docs nav. Thoughts?